.eslintrc.json

```diff
@@ -0,0 +1,32 @@
+{
+  "env": {
+    "browser": true,
+    "es6": true,
+    "es2017": true,
+    "es2020": true,
+    "commonjs": true,
+    "node": true
+  },
+  "extends": [
+    "eslint:recommended",
+    "plugin:ava/recommended",
+    "plugin:node/recommended-module",
+    "plugin:import/errors",
+    "plugin:import/warnings",
+    "google"
+  ],
+  "globals": {
+    "Atomics": "readonly",
+    "SharedArrayBuffer": "readonly"
+  },
+  "parserOptions": {
+    "ecmaVersion": 2020,
+    "sourceType": "module"
+  },
+  "rules": {
+    "max-len": "warn",
+    "no-invalid-this": "warn",
+    "camelcase": "warn",
+    "require-jsdoc": "warn"
```

Comment:
I see that this rule was deprecated: https://eslint.org/docs/rules/require-jsdoc

Reply:
This rule is still included within ESLint and was activated from `eslint-config-google`.

This corresponding line reduces the errors into warnings. If you do not want to show warnings for missing JSDoc then we can turn `require-jsdoc` to `off`.

Reply:
I see that ESLint [recommends](https://eslint.org/blog/2018/11/jsdoc-end-of-life) using the community supports [`eslint-plugin-jsdoc`](https://github.com/gajus/eslint-plugin-jsdoc)

Reply:
All right. Let's keep it then.